### PR TITLE
add user signup

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please review my pull request. Thanks po.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 class HomeController < ApplicationController
   def index
+    @user = current_user
+  end
+
+  def signup
```
Comment (MuhammadTamzid) [inline app/controllers/home_controller.rb]: RESTfull

Comment (dieunb) [inline app/controllers/home_controller.rb]: :+1:

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      login @user
+      flash[:success] = "Welcome to Framia eLearning system!"
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: i18n

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -1,4 +1,9 @@
 class HomeController < ApplicationController
   def index
+    @user = User.find_by id: session[:user_id]
+  end
+
+  def signup
```
Comment (dieunb) [inline app/controllers/home_controller.rb]: pls, follow tutorial to do signup task

**Pull request merged**